### PR TITLE
enhance: hawk response nonce checking

### PR DESCRIPTION
**Pull request by efahy**
…gging warnings

### CODE REVIEW

File: api_client/api_client.py [hunk 1]
```diff
@@ -68,6 +68,7 @@ def _get_hawk_sender(url, method, content_type, content):
             method=method,
             content=content,
             content_type=content_type,
+            seen_nonce=lambda x, y, z: False,
```
Comment (BenAndrew-Kainos) [inline api_client/api_client.py:71]: Add docstring to explain this

Comment (efahy) [reply api_client/api_client.py:75]: Done

Comment (andytaylorkainos) [reply api_client/api_client.py:75]: maybe worth adding that docstring to our perf tests client

Comment (andytaylorkainos) [reply api_client/api_client.py:75]: just saw that you already have 👍

Comment (efahy) [reply api_client/api_client.py:75]: Already done - Benny commented on it

**Pull request merged**